src/main/java/usecases/CrazyEights.java

```diff
@@ -33,17 +33,29 @@ public void startGame() {
             this.currPlayer = this.players[this.currPlayerIndex];
             Card card = null;
             String crd;
+            boolean looped = false;
+
+            this.controller.sendOutput(this.currPlayer.getName() + "'s Turn");
```

Comment:
Perhaps add dashes to the beginning and end of this string to section off parts of the console more clearly